src/LuaTransformer.ts

```diff
@@ -1148,8 +1147,7 @@ export class LuaTransformer {
 
     public transformSetAccessorDeclaration(
         setAccessor: ts.SetAccessorDeclaration,
-        className: tstl.Identifier,
-        classDeclaration: ts.ClassLikeDeclaration
```

Comment:
Are you sure you want to remove the declaration parameter? I think there could be use cases were you would want to access it.

Or is it possible to access the ClassDeclaration via the SetAccessorDeclaration ?

Reply:
setAccessor.parent should be the class declaration. I would also prefer to remove the className actually, but I'm not sure yet how exactly.